### ISSUE TITLE
QA data used in image-QA ablation study

### ISSUE THREAD

**Issue by j-min** [airsplay/lxmert#56]
Hi,

When you did ablation experiments about image-QA loss (1, 2 in table 4),
did you just use COCO-Cap and VG-Cap or did you still include VQA/GQA/VG-QA questions for other pretraining tasks? 

![image](https://user-images.githubusercontent.com/18069263/81023288-c1d2b580-8e24-11ea-80dc-3f4595f5a235.png)

![image](https://user-images.githubusercontent.com/18069263/81023329-d8790c80-8e24-11ea-92af-c6740cca044a.png)

**Comment by airsplay** [comment]
Thanks for pointing it out. We kept using the image-questioning data (VQA, GQA, VG QA) in other tasks when pre-training without image-QA loss.

**Comment by j-min** [comment]
Thank for quick reply! I think I finally found a bug in my implementation :)
